### PR TITLE
Input Dropdown: clear filter on select

### PR DESCRIPTION
**Pull request by tobias-schreiner**
#### Checklist
* [x] The contents of this PR are mine to submit. No third party code or other material is being committed.
* [x] New third party dependencies (if any) are linked via `package.json`. Third party dependencies are licensed under one of the following: Apache, MIT, BSD, Mozilla Public License, Eclipse Public License, or Oracle Binary Code License.
* [x] The code in this PR does not contain export-restricted encryption algorithms.

#### Ticket / Issue
Clear the filter text input of the input drop-down component after user selected an item

#### Description of Proposed Changes
Reset on selection

#### Documentation CI URL
https://pages.github.houston.softwaregrp.net/sepg-docs-qa/microfocusbbn_CI_UXAspects_clear-filter-on-select
https://pages.github.houston.softwaregrp.net/sepg-docs-qa/uigroupbbn_CI_ux-aspects-micro-focus_clear-filter-on-select/#/landing

#### Downstream Build(s)
https://jenkins.swinfra.net/job/SEPG/job/microfocusbbn/job/microfocusbbn%7EUXAspects%7Eclear-filter-on-select%7ECI/
https://jenkins.swinfra.net/job/SEPG/job/uigroupbbn/job/uigroupbbn%7Eux-aspects-micro-focus%7Eclear-filter-on-select%7ECI/

### CODE REVIEW

File: src/components/input-dropdown/input-dropdown.component.spec.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ describe('InputDropdownComponent', () => {
         spyOn(component.filterChange, 'emit');
         spyOn(component.filterInputElement.nativeElement, 'focus');
 
-        component.resetFilter(new MouseEvent('click'));
+        component.resetFilter();
```
Comment (donna-marijne) [inline src/components/input-dropdown/input-dropdown.component.spec.ts:41]: Probably this test should be modified, or a new test created, to verify that selecting an item clears the filter.

Comment (tobias-schreiner) [reply src/components/input-dropdown/input-dropdown.component.spec.ts:41]: I am not sure about the second build link. How can I create a build for ux-aspecs-micro-focus, which consumes my unmerged changes from the UXAspects repository?

Comment (donna-marijne) [reply src/components/input-dropdown/input-dropdown.component.spec.ts:41]: If you create a branch with the same name and create a build for that, the build will pick up this build artifact as the dependency.

**Comment by tobias-schreiner** [comment]
Sorry for that, I now have the new version. It does not seem to have my changes, though.

I created a branch with the same name as in the UXAspects repository, as you suggested, Does this really magically consume the changes from this repository? Or is there anything else to do?

**Pull request merged**